components/slider/index.css

```
@@ -455,17 +482,269 @@ governing permissions and limitations under the License.
 /* backwards compatibility: these are not required, but they make the handle go to the edegs and align right */
 .spectrum-Slider-handleContainer,
 .spectrum-Slider-trackContainer {
-  width: calc(100% + var(--spectrum-slider-handle-width));
+  width: calc(100% + var(--spectrum-slider-handle-size));
```

```suggestion
  inline-size: calc(100% + var(--spectrum-slider-handle-size));
```
Let's switch to the logical property for a bit of future-proofing.